hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/ConnectionContext.java

```diff
@@ -42,7 +43,10 @@
   private int numThreads = 0;
   /** If the connection is closed. */
   private boolean closed = false;
-
+  /** Last timestamp the connection was active. */
+  private long lastActiveTs = 0;
+  /** The connection's active status would expire after this window. */
+  private long activeWindow = 30 * 1000;
```

Comment:
Let's add the unit and probably do: activeWindowMs = TimeUnit.SECONDS.toMillis(30)